### PR TITLE
Fix container create model for volume mount

### PR DESCRIPTION
**Pull request by dsajanice**
Fix for https://github.com/Azure/iotedge/issues/147

The swagger generated model for volume mounting in container create options didn't match what edgeAgent was using resulting in user specified volumes not being mounted in containers. This is a regression from public preview bits and will be fixed with this PR.

### CODE REVIEW

File: edgelet/docker-rs/src/models/container_create_body.rs [hunk 1]
```diff
@@ -58,7 +58,7 @@ pub struct ContainerCreateBody {
     #[serde(rename = "Image", skip_serializing_if = "Option::is_none")]
     image: Option<String>,
     #[serde(rename = "Volumes", skip_serializing_if = "Option::is_none")]
-    volumes: Option<::models::ContainerConfigVolumes>,
+    volumes: Option<Value>,
```
Comment (varunpuranik) [inline edgelet/docker-rs/src/models/container_create_body.rs]: It feels like this should be Option<::std::collections::HashMap<String, Value>>

File: edgelet/edgelet-docker/tests/runtime.rs [hunk 2]
```diff
@@ -328,11 +331,13 @@ fn container_create_succeeds() {
...
     );
     let memory: i64 = 3221225472;
+    let volumes = json!({"/test1": {}, "/test2": {}});
     let create_options = ContainerCreateBody::new()
         .with_host_config(
             HostConfig::new()
                 .with_port_bindings(port_bindings)
-                .with_memory(memory),
+                .with_memory(memory)
+                .with_binds(vec!["/a:/a".to_string()]),
```
Comment (varunpuranik) [inline edgelet/edgelet-docker/tests/runtime.rs]: You are not asserting this anywhere looks like..

Comment (dsajanice) [reply edgelet/edgelet-docker/tests/runtime.rs]: Ah yes. I should remove this. This was orthogonal to my test.

File: edgelet/docker-rs/src/models/container_create_body.rs (hunk 1 above)
Comment (dsajanice) [reply edgelet/docker-rs/src/models/container_create_body.rs]: Yes, changed it.

**Comment by myagley** [comment]
Looks like you just need to run `cargo fmt`

**Pull request merged**